packages/blueprints-integration/src/rundown.ts

```diff
@@ -292,6 +299,13 @@ export interface IBlueprintPieceInstance<TMetadata = unknown> {
 
 	piece: IBlueprintPieceDB<TMetadata>
 
+	/** The time the system started playback of this part, null if not yet played back (milliseconds since epoch) */
+	startedPlayback?: Time
```

Comment:
Is the comment supposed to be "... _undefined_ if not yet played back" or is the time `Time | null`?